### PR TITLE
Issue #3216 update che templates

### LINKED ISSUE #3216: Che addon timeout when trying to run a workspace
### General information

  * Minishift version: v1.33.0+ba29431
  * OS: Linux
  * Hypervisor: KVM

### Steps to reproduce

  1. Start minishift
  2. Apply addon che `minishift addon apply che`
  3. Go to Che dashboard
  4. Choose e.g. Java workspace (but fails with Python, Nodejs, ...)
  5. Wait ~10mins.

It works fine with Che versions 6.x and with 7.0.0-beta-1.0. Does not work with latest, nightly, 7.0.0-beta-2.0 and 7.0.0-beta-3.0 (some of these are the same). 

### Actual

`pulling image "eclipse/ubuntu_jdk8"`
`Successfully pulled image "eclipse/ubuntu_jdk8"`
`Created container`
`Started container`
`Error: Failed to run the workspace: "Waiting for Kubernetes environment 'default' of the workspace'workspacexwikidqvvhifiumq' reached timeout"`

### PR DESCRIPTION
**Pull request by l0rd**
Updated che template with latest upstream one
Fixes #3216

### CODE REVIEW

**Comment by agajdosi** [comment]
ok to test

**Comment by praveenkumar** [comment]
retest this please

File: addons/che/templates/che-server-template.yaml [hunk 1]
```diff
@@ -38,6 +38,10 @@ objects:
       port: 8080
       protocol: TCP
       targetPort: 8080
+    - name: metrics
+      port: 8087
+      protocol: TCP
+      targetPort: 8087
```
Comment (praveenkumar) [inline addons/che/templates/che-server-template.yaml]: 👍

File: addons/che/templates/che-server-template.yaml [hunk 2]
```diff
@@ -254,6 +283,14 @@ parameters:
...
   description: Force an admin to update password after 1st login. True by default
   value: 'true'
+- name: CHE_KEYCLOAK_REALM
+  displayName: Identity provider Realm
+  description: Identity provider Realm. Defaults to che
+  value: 'che'
+- name: CHE_KEYCLOAK_CLIENT__ID
+  displayName: Identity provider Client ID
+  description: Identity provider Client ID. Defaults to che-public
+  value: 'che-public'
```
Comment (praveenkumar) [inline addons/che/templates/che-server-template.yaml]: 👍

**Pull request merged**